src/test/java/bsh/TestBshScriptEngine.java

```diff
@@ -41,11 +41,23 @@ public static void main( String [] args )
 		assertTrue( engine.get("bar").equals("gee") );
 		assertTrue( engine.eval("bar").equals("gee") );
 
+		// use reflection to pick available method
+		Method invokeMe = null;
+		try {
+			invokeMe = Invocable.class.getMethod( "invokeFunction", String.class, Object[].class );

```

Comment:
You probably won't need to do any of this. Just point to the method you want to use in script and it will probably work, automagically. The reflection stuffs are all collected in `bsh.Reflect` it's not the best and without need of improvement but I've sorted out several issues already so it will be best to continue there if you feel for have stuff to add.